### PR TITLE
17973 - Adding email component - reorg packages

### PR DESCRIPTION
**Pull request by hfekete**
17973 Added email component
- example of internationalization with i18n
- example of nuxt-vitest (remove nuxt/test-utils) as they do not go well together
- cleanup of eslint warns and errors
- cleanup of packages (moved all common packages to the workspace, left only the packages that have to be in app package.json)

### CODE REVIEW

**Review by kialj876 — COMMENTED** [review]
Nice work, just a couple comments / questions before we merge 👍

File: btr-web/btr-common-components/lang/en.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "labels": {
+    "emailAddress": "Email address: "
+  },
+  "errors": {
+    "validation": {
+      "invalidEmail": "Invalid email",
+      "sizeToLong": "Input to long"
```
Comment (kialj876) [inline btr-web/btr-common-components/lang/en.json:8]: *too

Comment (hfekete) [reply btr-web/btr-common-components/lang/en.json]: Thanks for spotting this

File: btr-web/btr-common-components/pages/examples/form.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!-- BCROS INPUT FIELDS component examples-->
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-common-components/pages/examples/form.vue:1]: yep

File: btr-web/btr-common-components/app.config.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export default defineAppConfig({
+  ui: {
+    input: {
+      rounded: 'rounded-none'
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-common-components/app.config.ts:4]: Yeah pretty handy

All the stuff, for example for input, from the preset part [input preset](https://ui.nuxt.com/forms/input#preset) can be overridden in the app.config.

**Review by kialj876 — APPROVED** [review]

**Pull request merged**